src/components/pages/upload/index.js

```diff
@@ -135,6 +138,16 @@ const Upload = props => {
 						<CardHeader>
 							<InputBox onChange={selectFiles} ref={inputRef} />
 							<AddFileButton
+
+                            // AddFileButton is disabled if there are already files selected ready to uploaded
+							disabled={
+								filenameCheck(filenames) !== filenames.length ||
+								filenames.length > 0	
+							}
```

Comment:
@susanX you don't need the first check `filenames.length > 0` is enough. The other check is for correct files and is attached to the upload button.